### ISSUE TITLE
EnzymeML Excel spreasheets don't open

### ISSUE THREAD

**Issue by danolson1** [EnzymeML/PyEnzyme#57]
I downloaded the example EnzymeML spreadsheets from the [Basic Example](https://github.com/EnzymeML/PyEnzyme/tree/main/examples/Basic%20example) folder, but I can't get them to open in Excel. I opened up the files in a text editor, and they don't look like Excel files, since they're in plain text (i.e. not zipped).

My computer is running Windows 10, and Excel for Microsoft 365 MSO (Version 2212 Build 16.0.15928.20196) 64-bit 

Can someone confirm whether the files are actually Excel files, and if not upload the correct ones?

Thanks,
Dan

**Comment by danolson1** [comment]
Ok. I figured it out. I needed to click on the file and then click the "view raw" link to download the file.

Note that one of the [EnzymeML spreadsheet links](http://api.enzymeml.org/template/download) on the EnzymeML website is down (https://enzymeml.org/tools/template/). The other link takes people to the github page, and might benefit from more explicit instructions for how to download the file.
